DeGruyter.js

```diff
@@ -275,19 +277,23 @@ var testCases = [
 						"firstName": "André"
 					}
 				],
-				"notes": [],
-				"tags": [],
-				"seeAlso": [],
-				"attachments": [],
-				"title": "Datenbanksysteme, Eine Einführung",
-				"publisher": "De Gruyter",
-				"place": "Berlin, Boston",
-				"ISBN": "978-3-486-59834-6",
-				"url": "http://www.degruyter.com/view/product/218879",
+				"date": "2015",
+				"ISBN": "9783110443752",
+				"abstractNote": "This textbook provides a systematic and thorough introduction to modern database systems. It presents today's leading relational technology in special detail. The 10th edition updates and expands the presentation of recent developments (i.e. main memory database systems and Big Data applications). Extensive examples illustrate the concepts, making the book especially suitable for self study.",
+				"extra": "10th expanded and updated edition",
```

Comment:
I don't know exactly. I looked at some more examples now and could only find examples with edition. However, the id "biblRemark" sounds for me more general than just edition. How about to store it in edition if this is still empty and otherwise stor it in a note?